app/styles/components/utils/social-media-handler.less

```diff
@@ -0,0 +1,55 @@
+.social-handle-container {
+  position: relative;
+
+  .upf-infinite-select {
+    max-width: 100%;
+    .padding-xx-sm;
+
+    .upf-infinite-select__item {
+      padding: 0;
+      .padding-top-xxx-sm;
+      .padding-bottom-xxx-sm;
+    }
+
+    .row-selected span {
+      .text-color-default;
+    }
+
+    .fx-basis-px-12 {
+      flex-basis: 12px;
+    }
```

Comment:
Already exist here https://github.com/upfluence/oss/blob/master/less/utilities/_flex.less#L31

Reply:
No, basis are not generated in pixels sadly.
I think I didn't add it in the _flex.less because we would not really need it.
It is helpful in this case though, that's why I added it in "local" less file